subsys/partition_manager/pm.yml.fmfu

```diff
@@ -0,0 +1,8 @@
+#include <autoconf.h>
+
+fmfu_storage:
+  placement:
+    before: [tfm_storage, end]
```

Comment:
I don't see that `tfm_storage` can be enabled to be put into external flash so I don't think it makes sense to have this here.

Reply:
@jayteemo This is my only I want this to be changed comment.